src/hyperloglog.c

```diff
@@ -1295,8 +1295,7 @@ void pfcountCommand(client *c) {
             hdr->card[5] = (card >> 40) & 0xff;
             hdr->card[6] = (card >> 48) & 0xff;
             hdr->card[7] = (card >> 56) & 0xff;
-            /* This is not considered a read-only command even if the
-             * data structure is not modified, since the cached value
+            /* This is considered a read-only command even if the cached value
```

Comment:
maybe the comment needs some representation next to where we do lookupKeyRead?
(we do have some comment in the command table explaining the flags).

in any case, i'd like to extend it to state the considerations and implications that we now discussed.
i.e. the command is flagged as read-only, and uses lookupKeyRead (which will not let it access an already expired value), but it does mark things to be propagated (as explained in the current comment).